tests/suites/test_suite_pk.data

```diff
@@ -138,18 +138,35 @@ Check pair #2 (EC, bad)
 depends_on:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED
 mbedtls_pk_check_pair:"data_files/ec_256_pub.pem":"data_files/server5.key":MBEDTLS_ERR_ECP_BAD_INPUT_DATA
 
-Check pair #3 (RSA, OK)
-depends_on:MBEDTLS_RSA_C
-mbedtls_pk_check_pair:"data_files/server1.pubkey":"data_files/server1.key":0
-
-Check pair #4 (RSA, bad)
-depends_on:MBEDTLS_RSA_C
-mbedtls_pk_check_pair:"data_files/server1.pubkey":"data_files/server2.key":MBEDTLS_ERR_RSA_KEY_CHECK_FAILED
-
+#Check pair #3 (RSA, OK)
+#depends_on:MBEDTLS_RSA_C
+#mbedtls_pk_check_pair:"data_files/server1.pubkey":"data_files/server1.key":0
+#
+#Check pair #4 (RSA, bad)
+#depends_on:MBEDTLS_RSA_C
+#mbedtls_pk_check_pair:"data_files/server1.pubkey":"data_files/server2.key":MBEDTLS_ERR_RSA_KEY_CHECK_FAILED
+#
```

Comment:
Why is this bit commented out? It shouldn't be affected by the new code. This PR shouldn't make any existing test fail. Please uncomment.

Reply:
Done